Elasticsearch Container does not wait until container is ready

**Describe the bug**

Elasticsearch container does not wait until container is ready. It breaks with a URLError.
```
E               urllib.error.URLError: <urlopen error [Errno 61] Connection refused>

../../../.asdf/installs/python/3.11.1/lib/python3.11/urllib/request.py:1351: URLError
------------------------------------------ Captured log setup -------------------------------------------
INFO     testcontainers.core.container:container.py:53 Pulling image elasticsearch:8.6.0
INFO     testcontainers.core.container:container.py:64 Container started: 8fcbabbc2cb5
```

**To Reproduce**

```python
    with ElasticSearchContainer("elasticsearch:8.6.0") as es:
        connection_url = es.get_url()
        yield es
```

**Runtime environment**

Tested on Macbook Pro M1.

--- a/elasticsearch/testcontainers/elasticsearch/__init__.py
+++ b/elasticsearch/testcontainers/elasticsearch/__init__.py
@@ -14,6 +14,7 @@ import logging
 import re
 import urllib
 from typing import Dict
+from urllib.error import URLError
 
 from testcontainers.core.container import DockerContainer
 from testcontainers.core.waiting_utils import wait_container_is_ready
@@ -83,7 +84,7 @@ class ElasticSearchContainer(DockerContainer):
         for key, value in _environment_by_version(major_version).items():
             self.with_env(key, value)
 
-    @wait_container_is_ready()
+    @wait_container_is_ready(URLError)
     def _connect(self) -> None:
         res = urllib.request.urlopen(self.get_url())
         if res.status != 200:

